### DISCUSSION TITLE
Saucedacity Name Change

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#38] [Polls]
As we start to take off, some are looking at our name. I realize that it's not the best name, so I'm thinking of changing it. However, that is up to you guys.

Feel free to voice thoughts in this discussion.

**Comment by poyribn** [comment]
with a silly name like Saucedacity, the software sounds like a toy or a meme, which implies it's fun or easy to use but also useless for advanced use
(easy-to-use is true, useless for advanced use is false).

but we also shouldn't have a name that's too serious. that would scare away beginners before they would even try the software.

but if we can't change the name, don't sweat it. eventually the people would know what Saucedacity means, and won't even think about the name's original connotations anymore. a great example of this is Audacity: that word means 'boldness, confidence, recklessness', but when talking about software, people don't think about this, but only about the audio editor (and the latest controversy).

**Comment by turtley12** [comment]
I think that if the name does end up changing, it should change when the first release after the qt-migration happens. At the moment saucedacity looks almost identical to audacity, and the name mostly makes sense. After the switch to qt saucedacity likely won't share as much code with audacity as it does now and it'll be more of a different thing, and I feel like a name change would make more sense.

**Reply by poyribn** [reply]
after the UI switches to using Qt it looks more modern, so we want people's first impression of "Tenacity" to have this new look, so we keep the "Saucedacity" name until the UI finished switching to Qt.
